src/phantom/base/typography/_bundle.scss

```diff
@@ -2,3 +2,15 @@
 @import "links_config";
 @import "lists_config";
 @import "paragraphs_config";
+
+@font-face {
+  font-family: "Calibre Light";
+  src: url('https://casper.com/static-assets/nightshade-core/src/typography/fonts/CalibreWeb-Light.woff2') format('woff2'),
```

Comment:
If we want to go this route we may want to use the version from the Rails CDN which is the other AWS instance. 

Another option is to create a separate bucket just for fonts. This could work well in the case of self-hosting our typography.com files. 

@jpsilvashy can you weigh in here...we want to store our font files in a spot where we can access them across our apps. the best option for now is to reference them directly from the server. should we create a new bucket on the casper classic amazon aws account or the one you set up?